docs/config.md

```diff
@@ -43,6 +43,305 @@ For `1x`/`2x`, initial learning rate decays by a factor of 10 at the 8/16th and
 For `20e`, initial learning rate decays by a factor of 10 at the 16th and 19th epochs.
 - `{dataset}`: dataset like `coco`, `cityscapes`, `voc_0712`, `wider_face`.
 
+## An Example of Mask R-CNN
+
+To help the users have a basic idea of a complete config and the modules in a modern detection system,
+we makea brief comments on the config of Mask R-CNN using ResNet50 and FPN as the following.
+For more detailed usage and the corresponding alternative for each modules, please refer the API documentation.
+
+```python
+model=dict(
+    type='MaskRCNN',  # The name of detector
+    pretrained='torchvision://resnet50',    # The ImageNet pretrained backbone to be loaded
+    backbone=dict(  # The config of backbone
+        type='ResNet',    # The type of the backbone
```

Comment:
We may refer each component to its docstring to help users find more details. E.g., for the line `type='ResNet'`, we may refer to `https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/backbones/resnet.py#L288`.